localconfig.env.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+# Flask config
+DEBUG=False
+IP=os.environ.get('GALLERY_IP', 'localhost')
+PORT=os.environ.get('GALLERY_PORT', '6969')
+SERVER_NAME = os.environ.get('GALLERY_SERVER_NAME', 'localhost:6969')
+SECRET_KEY = os.environ.get('GALLERY_SECRET_KEY', '')
+
+# LDAP config
+LDAP_URL=os.environ.get('GALLERY_LDAP_URL', 'ldaps://ldap.csh.rit.edu:636')
```

Comment:
@devinmatte We don't use ldap.csh anymore right? Pretty sure the csh_ldap package should do auto-discovery.

Reply:
Looking at the prod deployment of Gallery, we've removed the `LDAP_URL` and `LDAP_USER_OU` vars, so I'll remove them.